src/plugins/delaymon/delaymon.cpp

```diff
@@ -0,0 +1,86 @@
+#include "delaymon.h"
+
+#include <cmath>
+
+#define PLUGIN_NAME "[DELAYMON]"
+
+// Works on Windows 7
```

Comment:
Does this mean this only works for Win7 or that it only has been tested with Win7? If it's the latter, please make sure the plugin bails on startup when ran against the wrong version.

Reply:
This is copy-pasted comment is wrong.

Reply:
It must work everywhere

Reply:
If the comment is wrong then just remove it please

Reply:
Removed